pkg/mkimage-raw-efi/make-raw

```diff
@@ -70,21 +72,43 @@ IMGB_UUID=ad6871ee-31f9-4cf3-9e09-6f7a25c30053
 CONF_UUID=ad6871ee-31f9-4cf3-9e09-6f7a25c30054
 PERSIST_UUID=ad6871ee-31f9-4cf3-9e09-6f7a25c30059
 
+# content of rootfs partition
+ROOTFS_IMG=/parts/rootfs.img
+# content of conf partition
+CONF_FILE=/parts/config.img
+# content of persist partition
+PERSIST_FILE=/parts/persist.img
+# EFI boot directory
+EFI_DIR=/parts/EFI
+# early bootloader directory (optional)
+BOOT_DIR=/parts/boot
+# content of initrd installer image (optional)
+INITRD_IMG=/parts/initrd.img
+# content of installer ECO (optional)
+INSTALLER_IMG=/parts/installer.img
+# GRUB cfg override for our installer
+INSTALLER_GRUB_CFG=/parts/grub.cfg
+# IMX8 flash blob that needs to be written to image
+IMX8_BLOB=/parts/imx8-flash.bin
+# IMX8 configuration file with flashing parameters
+IMX8_CONF=/parts/imx8-flash.conf
+
```

Comment:
So with the current change, instead of statically define sizes for each image we calculate it?

Reply:
yes. not for everything, mostly rootfs